src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const population = [...document.getElementsByClassName('population')];
+const totalPopulationClass = document.querySelector('.total-population');
+const averagePopulationClass = document.querySelector('.average-population');
+
+const total = population.reduce((sum, currentPopulation) =>
+  sum + +currentPopulation.innerHTML.split(',').join(''), 0);
```

Comment:
Так не бажано переносити..
```suggestion
const total = population
  .reduce(
    (sum, currentPopulation) => sum + +currentPopulation.innerHTML.split(',').join(''),
    0
  );
```
Хоча й так не впевнений чи ок 😅